### PR TITLE
Feature/selling partner services

### PR DESCRIPTION
**Pull request by geni94**
## Users:
- CRUD actions
- DB write/read

## Authentication:
- JWT integration: decode/encode
- Bearer authentication in middleware

## Selling Partner actions
- some inventory and orders test calls

### CODE REVIEW

File: db/db.mjs [hunk 1]
```diff
@@ -7,13 +7,17 @@ const dbName = process.env.DB_NAME;
 const cloudUri = `mongodb+srv://eugenDb:${password}@cluster0.grdmy.mongodb.net/${dbName}?retryWrites=true&w=majority`;
```
Comment (lincolnaleixo) [inline db/db.mjs:7]: better put the "eugenDb" as env as well

File: index.js [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
+import userRoutes from './routes/users.js';
 
 dotenv.config();
 
+// start Express app
+const app = express();
+const port = 3000;
+
+// DB & Mongoose:
+const cloudUri = `mongodb+srv://eugenDb:${process.env.DB_PASSWORD}@cluster0.grdmy.mongodb.net/${process.env.DB_NAME}?retryWrites=true&w=majority`;
```
Comment (lincolnaleixo) [inline index.js:17]: same here (eugenDb)

**Review by lincolnaleixo — CHANGES_REQUESTED** [review]
please check comments about the db string

File: index.js (hunk 2 above)
Comment (geni94) [reply index.js]: Fixed 1eadeac7dee79530909873c65efa1b4698ae3c72

File: db/db.mjs (hunk 1 above)
Comment (geni94) [reply db/db.mjs]: fixed 1eadeac7dee79530909873c65efa1b4698ae3c72

**Review by lincolnaleixo — APPROVED** [review]

**Pull request merged**